packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -253,6 +253,8 @@ export const components = ({
             justifyContent: "center",
             alignItems: "center",
             borderRadius: 0,
+            borderWidth: 0,
+            borderBottomWidth: "1px",
```

Comment:
Added this since its related to the contrast work and visual alignment with Figma (as of last week).

Reply:
@benlister-okta The banner only has a border on the bottom?

Reply:
That's correct. Alex and I discussed and went this direction since in the new template the guidance is (or will be) to use banner at the top of the browser. And for existing instances, we are ok with the look since it adds subtle contrast but still fits the system.

Reply:
Is rendering this banner at the top of the browser/above the top nav even possible?

Reply:
What I mean is, I thought the nav components will be injected into their apps. They may not have the ability to render anything at a higher level than the top nav

Reply:
That makes sense, I am not sure how it will be implemented globally, it may need to be done at a feature level to support that. However, we tested it with the bottom-only border in the app and were fine with the look so it can be something we work towards in the future.

Reply:
We may need to run this up the chain to determine if it is a breaking visual change or not. I know it's pretty minor but not sure how strict we need to be with stuff like this

Reply:
I think it definitely warrants a message along with the release at very least. The border version was added 2 releases ago so its still a "new" change but we should make sure were letting people know.